util/stmtsummary/reader.go

```diff
@@ -313,11 +313,19 @@ var columnValueFactoryMap = map[string]columnValueFactory{
 	ClusterTableInstanceColumnNameStr: func(reader *stmtSummaryReader, ssElement *stmtSummaryByDigestElement, ssbd *stmtSummaryByDigest) interface{} {
 		return reader.instanceAddr
 	},
-	SummaryBeginTimeStr: func(_ *stmtSummaryReader, ssElement *stmtSummaryByDigestElement, _ *stmtSummaryByDigest) interface{} {
-		return types.NewTime(types.FromGoTime(time.Unix(ssElement.beginTime, 0)), mysql.TypeTimestamp, 0)
+	SummaryBeginTimeStr: func(reader *stmtSummaryReader, ssElement *stmtSummaryByDigestElement, _ *stmtSummaryByDigest) interface{} {
+		beginTime := time.Unix(ssElement.beginTime, 0)
+		if beginTime.Location() != reader.tz {
+			beginTime = beginTime.In(reader.tz)
+		}
```

Comment:
The test case doesn't prove that the beginTime and endTime follow the time zone.